src/helm/env.d/dev/values.impress.yaml.gotmpl

```diff
@@ -50,6 +51,8 @@ backend:
     AWS_S3_SECRET_ACCESS_KEY: password
     AWS_STORAGE_BUCKET_NAME: impress-media-storage
     STORAGES_STATICFILES_BACKEND: django.contrib.staticfiles.storage.StaticFilesStorage
+    CONVERSION_API_URL: http://impress-y-provider:443/api/convert-markdown/
```

Comment:
There is no flexibility with this approach, we will have to create a env var for every endpoints if we do that.
Like propose in PM, `YJS_SERVER_API` will fit more what it is.

Reply:
Yes discussed IRL. I agree with you. However, at the time we discussed my endpoint, I add no idea if you would add more endpoint. Also, the express server isn't really configured to use a specific base URL … 

I can refactor this part, but I feel I am solving future problems, which wasn't the top priority while working on my urgent topics. This won't be a huge refactoring, I'll try to do it.

Reply:
the microservice is name `y-provider`, you suggest `yjs_server`, actually all the code of this microservice is run by a `collaboration server` … IDK what really should be the naming of this var. However, I will follow your suggestions, but I don't want my call the microservices through the ingress

Reply:
updated !